database-mysql/index.js

```diff
@@ -73,7 +76,8 @@ const clearDb = function (callback) {
 };
 
 module.exports = {
-  updateDb,
+  insertProducts,
+  insertReviews,
```

Comment:
Good separation of concerns using two functions instead of a general updateDB functions.